packages/trpc/server/routers/viewer/workflows/getByViewer.handler.ts

```diff
@@ -94,20 +99,45 @@ export const getByViewerHandler = async ({ ctx }: GetByViewerOptions) => {
     },
   });
 
+  const teamMemberships = user.teams.map((membership) => ({
```

Comment:
same here (changes of this file are pretty much the same as we have it in `getByViewer.handler` for event types)